### PR TITLE
Fix self update does not run command

### PR DESCRIPTION
**Pull request by luphaz**
As suggested [here](https://github.com/heetch/s3update/issues/8) the command call has been replaced

I tried the fix locally with `aww`, for more information
please refer to [this issue](https://retool.atlassian.net/browse/DELIVERY-213)

### CODE REVIEW

File: s3update.go [hunk 1]
```diff
@@ -160,25 +159,8 @@ func runAutoUpdate(u Updater) error {
...
-			// error is of type *ExitError. Other error types may be
-			// returned for I/O problems.
-			if exiterr, ok := err.(*exec.ExitError); ok {
-				// The command didn't complete correctly.
-				// Exiting while keeping the status code.
-				os.Exit(exiterr.Sys().(syscall.WaitStatus).ExitStatus())
-			} else {
-				return err
-			}
+		if err := syscall.Exec(os.Args[0], os.Args, os.Environ()); err != nil {
```
Comment (Charrette) [inline s3update.go]: Shouldn't the second arg be `os.Args[1:]`?
I never used it so, not sure

Comment (luphaz) [reply s3update.go]: I do not know why but the second argument as to be the whole `os.Args` array, not the one without the exec.

I supposed it's necessary to properly replace the process 🤷‍♂ 

Tried locally both version, the one using `os.Args[1:]` was failing.

According to both documentation and method signature it seems consistent at least : 
* https://golang.org/src/syscall/exec_unix.go?s=7396:7461#L252
* https://golang.org/src/os/exec/exec.go?s=6250:6295#L158

Comment (Charrette) [reply s3update.go]: Ok, good to know! Thanks

**Review by WinterNis — APPROVED** [review]

File: s3update.go (hunk 1 above)
Comment (luphaz) [reply s3update.go]: Is it ok for you ? You did not approved the review yet 😄 🤔

Comment (Charrette) [reply s3update.go]: oops sorry :D

**Review by Charrette — APPROVED** [review]

File: s3update.go (hunk 1 above)
Comment (luphaz) [reply s3update.go]: No pb 😄 I was just wondering if something else catches your eyes

**Pull request merged**